### DISCUSSION TITLE
Using lastversion for downloading a single file (name variable) from topmost release that include it

### DISCUSSION THREAD

**Discussion by DiMarco1976** [dvershinin/lastversion#101] [Q&A] [answered]
Hello,

I would like always to download the portable x64-file from the newest (pre-) release of per example PeaZip (within a batch-file). 
The current PeaZip release is PeaZip 9.2.0, but there exists a hidden tag 9.9.0 (only source files included) that you only find by comparing tags. I try to download the file "peazip_portable-9.2.0.WIN64.zip" of the 9.2.0 release. 

When I use 
`lastversion peazip/PeaZip --assets -d (without --pre)` 
I get the ZIP source file of Tag 9.9.0.
When I try:
`lastversion peazip/PeaZip --format assets --filter WIN64.zip -d PortablePeaZip_x64.zip`
I get: nothing

What I really want from lastversion:
1. Browse the peazip/PeaZip pre-releases and releases on Github
2. Get the latest pre-release or release in which the the file "peazip_portable-<WIN64.zip" is included by ignoring all tags in the asset is not included. "<" stands for the unknown version number in the middle of the file, Can I use "<" when I have two known parts of a file when the middle part is the unknown version number? In the PeaZip case the "<" is not needed because only one file ends up with WIN64.zip, but it would be a great feature. 
3. Download that file and rename it to "PortablePeaZip_x64.zip"

I would be really glad archieving it using lastversion.

Regards,
DiMarco76

**Comment by dvershinin** [comment, marked as answer]
Here's a command for you: 

```bash
lastversion peazip/PeaZip --having-asset --pre --format assets --filter WIN64.zip -d PortablePeaZip_x64.zip
```

**Reply by DiMarco1976** [reply]
Thank you very much  ;-) I think I misunderstood the --having-asset switch.
